atlas/schema.hcl

```diff
@@ -352,7 +352,7 @@ table "user_snapshot_v1" {
   comment = "A snapshot in time of a user profile. A new snapshot is generated everytime a user modifies information related to their profile."
   column "user_id" {
     null = false
-    type = bigint
+    type = text
```

Comment:
i think there's a uuid datatype please verify

Reply:
![image](https://github.com/sunb26/xat/assets/56877739/8d5c3c67-724e-43c8-b509-d7ed200d71d5)

Yes there is thanks for catching that. Is it going to be generated by Clerk or the BE or should we default generate it from the SQL?

Reply:
It's gonna be from clerk